pkg/controllers/deploy_ctrl.go

```diff
@@ -50,6 +50,48 @@ func (c *DeployControllerImpl) Reconcile(ctx context.Context, mc v1beta1.Milvus,
 		return errors.Wrap(err, "update milvus rolling mode status")
 	}
 	biz := c.bizFactory.GetBiz(component)
+	// Reconcile QueryNode deploy mode
+	if component == QueryNode {
```

Comment:
you may move the code inside to a function named like ReconcileQueryNodeDeployMode() to avoid making `Reconcile()` too long.